tests/integration/test_server_reload/test.py

```diff
@@ -246,8 +246,11 @@ def test_change_grpc_port(cluster, zk):
             grpc._channel._InactiveRpcError, match="StatusCode.UNAVAILABLE"
         ):
             grpc_query(grpc_channel, "SELECT 1")
+        grpc_channel.close()
```

Comment:
what about making `get_grpc_channel` a context manager and using it as `with get_grpc_channel(cluster, port=9100) as grpc_channel:`

It should look like `default_client` in the same file

Reply:
I'm okay with that, but I would first like to merge this simple solution ASAP to see if it's the cause of the problems.